app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -57,7 +56,7 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
   private val oppiaLogger: OppiaLogger,
   private val headerViewModelProvider: ViewModelProvider<NavigationDrawerHeaderViewModel>,
   private val footerViewModelProvider: ViewModelProvider<NavigationDrawerFooterViewModel>,
-  private val developerOptionsStarter: Optional<DeveloperOptionsStarter>
```

Comment:
Won't this cause build errors if in prod builds of the app? I'm fairly certain we need to keep this.

Can you try using the Guava Optional, instead, to see if it builds? That's a drop-in replacement in Dagger for Java 8 Optional.